src/answers-umd.js

```diff
@@ -509,9 +509,11 @@ class Answers {
    * @param {string} translations The translations, or a stringified JSON of possible translations
    * @param {Object} interpolationParams Params to use during interpolation
    * @param {number} count The count associated with the pluralization
+   * @returns {SafeString} The translation as a Handlebars SafeString
    */
   translateJS (translations, interpolationParams, count) {
-    return Translator.translate(translations, interpolationParams, count);
+    const translation = Translator.translate(translations, interpolationParams, count);
+    return this.renderer.SafeString(translation);
```

Comment:
Could you explain a bit why we need this here as well? I think `translateJS` will primarily be used with Javascript source, where we wouldn't be worried about escaping HTML.

Reply:
I added this to support building intermixed markdown inside of the javascript and passing it to the template to render. Not supporting intermixed markdown with the JS helper makes a lot of sense though! I can remove this